### DISCUSSION TITLE
What kind of GitHub Token is meant here?

### DISCUSSION THREAD

**Discussion by eXpl0it3r** [uraimo/run-on-arch-action#147] [Q&A] [answered]
I guess this might be obvious to everyone, but I'm a bit confused what kind of token you mean with "GitHub Token". Is this an access token? Or is there some publish package token one can create?

The documentation only talks about "GitHub Token" without mentioning what kind of token is meant or where to retrieve it.

**Comment by martin-g** [comment, marked as answer]
https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Usage: https://github.com/lh3/bwa/blob/79b230de48c74156f9d3c26795a360fc5a2d5d3b/.github/workflows/ci.yaml#L38

**Reply by eXpl0it3r** [reply]
Thank you! 🙂 

I've created a PR to add that link in the README, so it may be clearer for others as well: https://github.com/uraimo/run-on-arch-action/pull/148
